kafka-ui-react-app/src/components/Topics/List/ListItem.tsx

```diff
@@ -66,34 +66,36 @@ const ListItem: React.FC<ListItemProps> = ({
           {internal ? 'Internal' : 'External'}
         </div>
       </td>
-      <td>
-        <div className="has-text-right">
-          <Dropdown
-            label={
-              <span className="icon">
-                <i className="fas fa-cog" />
-              </span>
-            }
-            right
-          >
-            <DropdownItem onClick={clearTopicMessagesHandler}>
-              <span className="has-text-danger">Clear Messages</span>
-            </DropdownItem>
-            <DropdownItem
-              onClick={() => setDeleteTopicConfirmationVisible(true)}
+      {!internal ? (
+        <td className="value">
```

Comment:
This behaviour makes html of the table invalid. Please apply `!internal` to child element of last td in a row